README.md

```diff
@@ -22,6 +27,8 @@ Download to plugins/ColorByType and change the type to `ColorByType/View/Track/C
       },
 ```
 
+You can also do this when using **flatfile-to-json.pl** with **--trackType ColorByType/View/Track/ColorByTypeDraggable** flag.
```

Comment:
Need to add more description about which situation can use this flag.

Reply:
What king of description ? You can supplement it, though. I wrote this totally based on the description you told me.